s4e-web/src/app/views/settings/settings.component.html

```diff
@@ -1,6 +1,10 @@
 <div class="container">
   <nav class="navigation">
     <ul class="navigation__list">
+      <li *ngIf="showInstitutions$ | async">
+        <a href="javascript:void(0)" (click)="openParentInstitutionModal()">Dashboard</a>
```

Comment:
I know its temporary, but maybe change the name of the link to `show institution modal` or something